controllers/managedclusterForCGU_controller.go

```diff
@@ -292,10 +292,21 @@ func (r *ManagedClusterForCguReconciler) SetupWithManager(mgr ctrl.Manager) erro
 				UpdateFunc: func(e event.UpdateEvent) bool {
```

Comment:
Not sure. Do you think there is absolutely no way to miss an event?

Reply:
Not 100% sure either but at least I don't see any bug raised regarding missing events. Perhaps we just keep it as-is then.

Reply:
We need the create events when the controller is restarted. Updates to managed clusters when it's down won't get reconciled if we don't handle create events.

Reply:
I removed the requeue for not ready clusters.

Reply:
I am thinking another case - the cluster is changed back to False/Unknown after being requeued? May be rare, but in the case, we will need to requeue.

Reply:
> I am thinking another case - the cluster is changed back to False/Unknown after being requeued? May be rare, but in this case, we will need to requeue.
Confirmed with Jun. There is no need to requeue here as it will be re-queued during the next update event when the status changes to True.